Maquetado/views/login.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>Login</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="../application.css">
+</head>
+
+<body>
+  <div class='pageContainer'>
+    <div class='loginContainer'>
+      <img src='../resources/img/LogoWolox.png' class='woloxLoginLogo' alt='Wolox Logo Image' />
+      <div class='inputsContainer'>
```

Comment:
Because this is a form, please use a better element.
```suggestion
      <form class='inputsContainer'>
```